lib/Sema/TypeCheckAttr.cpp

```diff
@@ -2464,7 +2464,6 @@ void AttributeChecker::visitImplementsAttr(ImplementsAttr *attr) {
 void AttributeChecker::visitFrozenAttr(FrozenAttr *attr) {
   if (auto *ED = dyn_cast<EnumDecl>(D)) {
     if (!ED->getModuleContext()->isResilient()) {
-      diagnoseAndRemoveAttr(attr, diag::enum_frozen_nonresilient, attr);
```

Comment:
Maybe make this set the attribute to be invalid?

```suggestion
       attr->setInvalid()
```

Reply:
Added (I don't know what effect this has though 🙃)

Reply:
I hope it won't become a warning by default though? But if it did I guess the tests would catch it

Reply:
This effectively "removes" the attribute (it's what `diagnoseAndRemoveAttr` does, just without the diagnose part)